index.js

```diff
@@ -42,12 +42,17 @@ module.exports.pitch = function(request) {
 		if(err) return callback(err);
 		if (entries[0]) {
 			var workerFile = entries[0].files[0];
-			var constructor = "new Worker(__webpack_public_path__ + " + JSON.stringify(workerFile) + ")";
-			if(query.inline) {
-				constructor = "require(" + JSON.stringify("!!" + path.join(__dirname, "createInlineWorker.js")) + ")(" +
+			var constructor
+			if(query.service) {
+				constructor = "navigator.serviceWorker.register(__webpack_public_path__ + " + JSON.stringify(workerFile) + ", options);"

```

Comment:
Can you add detection here? Worker is pretty much defined everywhere, serviceWorker isn't

maybe something like
`navigator.serviceWorker ? navigator.serviceWorker.register(__webpack_public_path__ + " + JSON.stringify(workerFile) + ", options) : Promise.reject('No support')`